mmpose/apis/inference_tracking.py

```diff
@@ -0,0 +1,196 @@
+import numpy as np
+
+
+def iou(boxA, boxB):
+    """Compute the Intersection over Union (IoU) between two boxes .
+
+    Args:
+        boxA (list): The first box info (left, top, right, bottom, score).
+        boxB (list): The second box info (left, top, right, bottom, score).
+
+    Returns:
+        float: The IoU value.
+    """
+
+    xA = max(boxA[0], boxB[0])
+    yA = max(boxA[1], boxB[1])
+    xB = min(boxA[2], boxB[2])
+    yB = min(boxA[3], boxB[3])
+
+    inter_area = max(0, xB - xA + 1) * max(0, yB - yA + 1)
+
+    boxA_area = (boxA[2] - boxA[0] + 1) * (boxA[3] - boxA[1] + 1)
+    boxB_area = (boxB[2] - boxB[0] + 1) * (boxB[3] - boxB[1] + 1)
+
+    iou = inter_area / float(boxA_area + boxB_area - inter_area)
```

Comment:
Any chance that the denominator equals 0?

Reply:
Will this function be used elsewhere?

Reply:
Do not worry as boxA_area >= inter_area & boxB_area >= inter_area. The denominator equals 0 only is boxA_area = boxA_area = 0, which is impossible for mmdet to predict such a person bbox.

Reply:
Do we need assertion that `boxA[2] > boxA[0]` and `boxA[3] > boxA[1]` ? @innerlee

Reply:
If a zero-area bbox be considered an error, we can add assertions.
If it is allowed, then we may add a `max(1, blablha...)` at the denominator.

BTW what's the difference between `box` and `bbox`?

Reply:
I think adding assertions should be better. And one suggestion is that the previous steps may also consider this kind of error (box[2] <= box[0]). In detail, if such a bbox is given, function inference_top_down_pose_model may have some problems. We may also consider to add assertions in function _xyxy2xywh or anywhere suitable.

As for the difference box and bbox, function _compute_iou is more general and can compute iou for any two boxes. I use box because I do not want to emphasize they are person bounding boxes. However, we may not consider other kinds of boxes and I will use bbox to avoid confusion.

If you gays agree with my opinions, I will make these two changes very soon. @innerlee @jin-s13

Reply:
More checks are good. One thing to take care is that we 
* use assertion only when the it is an error, because a violation would stop the running program, and any unsaved computations would got lost. 
* If the case is not critical, then a warning is good enough. 
* If it is neglectable, we can simply ignore it and replace with safe neutral values

I vote for bbox.